example_test.go

```diff
@@ -252,3 +280,24 @@ func ExampleCopy_localToRemote() {
 	// Output:
 	// sha256:6f2590d54af17afaca41a6243e3c01b368117d24b32e7581a6dee1d856dd3c4b
 }
+
+func ExampleCopy_artifactManifestRemoteToLocal() {
+	src, err := remote.NewRepository(fmt.Sprintf("%s/%s", remoteHost, "source"))
+	if err != nil {
+		panic(err)
+	}
+	dst := memory.New()
+
+	ctx := context.Background()
+	err = oras.CopyGraph(ctx, src, dst, exampleSignatureManifestDescriptor, oras.DefaultCopyGraphOptions)
```

Comment:
Should we use `Copy`, `ExtendedCopy`, `CopyGraph` or `ExtendedCopyGraph` here?

Reply:
I think we can resolve by a given digest, and then call `CopyGraph()`.

Reply:
Changed accordingly.